db/migrate/20150609191815_create_songs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSongs < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp'
```

Comment:
Are you certain you need to enable this again?

The `schema.db` in `master` should already account for this:

```
ActiveRecord::Schema.define(version: 20150609174024) do

  # These are extensions that must be enabled in order to support this database
  enable_extension "plpgsql"
  enable_extension "uuid-ossp"
```

... so, no need to enable it in each migration.